dita/password-managers.dita

```diff
@@ -3,95 +3,50 @@
+    <topic id="guidance">
+      <title>Guidance</title>
       <body>
-        <p>The following table shows when you might use a password manager or vault:</p>
-        <table>
-          <tgroup cols="3">
-            <colspec colname="col1" colnum="1" colwidth="1*"/>
-            <colspec colname="col2" colnum="2" colwidth="1*"/>
-            <colspec colname="col3" colnum="3" colwidth="3*"/>
-            <thead>
-              <row>
-                <entry colname="col1">Scenario</entry>
-                <entry colname="col2">Tool</entry>
-                <entry colname="col3">Notes</entry>
-              </row>
-            </thead>
-            <tbody>
-              <row>
-                <entry colname="col1">Single user, personal accounts</entry>
-                <entry colname="col2">Password manager</entry>
-                <entry colname="col3">For accounts that only you use, or have access to, then you would probably store the details in a password manager. An example would be storing the username and password for your work email account; only you should have access.</entry>
-              </row>
-              <row>
-                <entry colname="col1">Multiple users, shared accounts</entry>
-                <entry colname="col2">Password manager or password vault</entry>
-                <entry colname="col3">Some accounts might be shared between a group of users. For example, a team might need to know the password for an encrypted document. If the access required is for a sensitive or operational system, then a more heavily protected tool such as a password vault might be appropriate.</entry>
-              </row>
-              <row>
-                <entry colname="col1">System access, no human use</entry>
-                <entry colname="col2">Password vault</entry>
-                <entry colname="col3">Some <ph conref="conrefs.dita#conrefs/moj"/> systems need to 'talk' directly to other systems. No humans are involved in the conversation. The passwords protecting these communications can - and should - be extremely complex. A strongly secured password vault would be ideal for this purpose.</entry>
-              </row>
-            </tbody>
-          </tgroup>
-        </table>
+      <p>Password managers come in several forms, for example, including standalone applications and those integrated into browsers or operating system.</p>
+      <p>The MoJ provides a built-in browser password manager for use on supported browsers. When logging into a website, the browser will prompt you to save your credentials securely.</p>
+      <p>Additionally, there are a small number of specific MoJ teams that have access to the <xref href="https://intranet.justice.gov.uk/guidance/security/equipment/equipment-management/user-access/passwords/" format="html" scope="internal">password manager 1Password for their work.</xref></p>
```

Comment:
This link doesn't make contextual sense here? I would suggest removing this line, or saying "Additionally, some specialist teams may have other dedicated approved password management tools for their use. Consult local advice and guidance for details on these".